sdk/translation/Azure.AI.Translation.Text/README.md

```diff
@@ -132,7 +132,7 @@ try
 
     Response<IReadOnlyList<TranslatedTextItem>> response = await client.TranslateAsync(targetLanguage, inputText).ConfigureAwait(false);
     IReadOnlyList<TranslatedTextItem> translations = response.Value;
-    TranslatedTextElement translation = translations.FirstOrDefault();
+    TranslatedTextItem translation = translations.FirstOrDefault();
```

Comment:
If you could move these samples over to C# files like we do in the rest of our libraries, that would be awesome, and it would help you detect issues like these because those would get built and run as part of our pipelines! If there's urgency to release, please file an issue to track this work.

Reply:
We should already have a work item for that in our backlog. I do plan to create issues in GH to track for you as well.